torchaudio/prototype/rnnt_loss.py

```diff
@@ -166,17 +87,20 @@ def rnnt_loss(
             False  # softmax needs the original logits value
         )
 
-    cost = _RNNT.apply(
-        logits,
-        targets,
-        logit_lengths,
-        target_lengths,
-        blank,
-        clamp,
-        fused_log_softmax,
-        reuse_logits_for_grads,
-    )
-    return cost
+    if blank < 0:  # reinterpret blank index if blank < 0.
+        blank = logits.shape[-1] + blank
```

Comment:
is this also needed for `alphas` and `betas`?

Reply:
It seems likely that this may be the case, but will leave this as a followup. Per offline discussion, it may be a better idea to just remove these functions which are not used internally and also not exposed to the user.